lib/rubygems/gemcutter_utilities.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 require_relative 'remote_fetcher'
 require_relative 'text'
+require 'json'
```

Comment:
I'm not sure it is possible to rely on `json` presence. It is already required at S3 signer. But for rubygems.org response there is already `YAML` endpoint used. I think we can try to keep it the same here.

https://github.com/rubygems/rubygems/blob/bec5fdf524c0ccd3d3e561db11455e4844049497/lib/rubygems/commands/owner_command.rb#L64-L79